book/modules/ROOT/pages/introduction.adoc

```diff
@@ -7,27 +7,39 @@
 
 Welcome to The Examples Book! This book contains a collection of information and examples that can be used to reinforce topics learned in https://datamine.purdue.edu[The Data Mine] seminar at https://purdue.edu[Purdue University]. The Examples Book is open to anyone! Even if you aren't a student at Purdue we hope you find helpful content here. 
 
-This book is broken down into "core" topics. These topics are listed in the options below. They range from getting started with data science to information for the corporate partners who want to sponsor a project with us. 
-
-In addition, the live version of the "core" book contains the most up-to-date slate of projects students in The Data Mine are working on. All projects from previous semesters live in the xref:projects:ROOT:introduction.adoc[Projects Archive] appendix and are open to the public.
-
 [TIP]
```

Comment:
Overall, perhaps we could change the "Check out `something`" to "Check out [`something`](google.com)", i.e. add links to the same places the buttons go? Or to the anchor link for the section (if for example we are talking about "Course Links" where it is an entire section. Maybe?